openshift/templates/clamav/clamav.bc.json

```diff
@@ -0,0 +1,97 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "clamav",
+    "creationTimestamp": null
+  },
+  "objects": [
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}"
+      }
+    },
+    {
+      "kind": "BuildConfig",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "${NAME}-build",
+        "creationTimestamp": null,
+        "labels": {
+          "app": "${NAME}"
+        }
+      },
+      "spec": {
+        "runPolicy": "Serial",
+        "completionDeadlineSeconds": 1800,
+        "triggers": [
+          {
+            "type": "ImageChange"
+          },
+          {
+            "type": "ImageChange",
+            "imageChange": {
+              "from": {
+                "kind": "ImageStreamTag",
+                "name": "${NAME}:latest"
+              }
+            }
+          }
+        ],
+        "source": {
+          "type": "Git",
+          "git": {
+            "uri": "${GIT_SOURCE_URL}"
+          }
+        },
+        "strategy": {
+          "type": "Docker",
+          "dockerStrategy": {
+            "from": {
+                "kind": "ImageStreamTag",
+                "name": "base-centos7:latest",
+                "namespace": "openshift"
+            }
+          }
+        },
+        "output": {
+          "to": {
+            "kind": "ImageStreamTag",
+            "name": "${NAME}:latest"
+          }
+        },
+        "resources": {
+          "requests": {
+            "cpu": "100m",
+            "memory": "2Gi"
+          },
+          "limits": {
+            "cpu": "1",
+            "memory": "4Gi"
+          }
+        }
+      },
+      "status": {
+        "lastVersion": 0
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "NAME",
+      "displayName": "Name",
+      "description": "The name assigned to all of the objects defined in this template.",
+      "required": true,
+      "value": "clamav"
+    },
+    {
+      "name": "GIT_SOURCE_URL",
+      "displayName": "GIT Source Repo URL",
+      "description": "A GIT URL to your source code.",
+      "required": true,
+      "value": "https://github.com/marklise/clamav.git"
```

Comment:
Can we point this default at  https://github.com/bcgov/clamav instead of Mark's?

Reply:
Update to bcgov version.